### ISSUE TITLE
C4 nephritic factor

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7518]
**Preferred term label:**
C4 nephritic factor

**Synonyms**
C4NeFs, C4NeF, Complement 4 nephritic factor

**Definition (free text, please give PubMed ID)**
These autoantibodies recognize C4b2a, the C3 convertase of the classical pathway of complement.
PMID: 28838767, PMID: 27146825, PMID: 23117396

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by MaudDLVT** [comment]
@julienN75

**Comment by pnrobinson** [comment]
added
